meteor/server/api/ingest/packageInfo.ts

```diff
@@ -77,6 +77,14 @@ async function onUpdatedPackageInfoForRundown(
 		)
 		return
 	}
+	if (tmpRundown.orphaned) {
```

Comment:
Does `orphaned` always mean no IngestRundown? I would expect that when it was manually unsynced or removed from NRCS during playback that IngestRundown should still be there?

Reply:
yes this is true, I'll remove this